src/test/kotlin/no/nav/syfo/service/MerVeiledningVarselFinderSpek.kt

```diff
@@ -92,6 +97,32 @@ object MerVeiledningVarselFinderSpek : Spek({
 
             varslerToSendToday.size shouldBeEqualTo 0
         }
+
+        it("Should send MER_VEILEDNING when user is under 67") {
+            coEvery { pdlConsumerMockk.isBrukerYngreEnn67(any()) } returns true
+            coEvery { sykmeldingServiceMockk.isPersonSykmeldtPaDato(LocalDate.now(), arbeidstakerFnr1) } returns true
+            embeddedDatabase.storeUtsendtVarsel(getUtsendtVarselToStore(LocalDateTime.now().minusMonths(5)))
```

Comment:
Jeg synes ikke man trenger å lagre et utsendt varsel her. Det er jo ikke en forutsetning for å få sendt varsel.